FAQ.md

```diff
@@ -54,10 +54,18 @@ Essentially, anything which provides the full identity of what you want to simul
 
 If you are unable to provide a molecule in the formats recommended above and want to attempt to infer the bond orders and atomic formal charges, there are tools available elsewhere that can provide guesses for this problem. These tools are not perfect, and the inference problem itself is poorly defined, so you should review each output closely (see our [Core Concepts](users/concepts) for an explanation of what information is needed to construct an OpenFF Molecule). Some tools we know of include:
 
-- the OpenEye toolkits' [`OEPerceiveBondOrders`](https://docs.eyesopen.com/toolkits/python/oechemtk/OEChemFunctions/OEPerceiveBondOrders.html) functionality
+- the OpenEye Toolkits' [`OEPerceiveBondOrders`](https://docs.eyesopen.com/toolkits/python/oechemtk/OEChemFunctions/OEPerceiveBondOrders.html) functionality
 - [MDAnalysis' RDKit converter](https://docs.mdanalysis.org/stable/documentation_pages/converters/RDKit.html?highlight=rdkit#module-MDAnalysis.converters.RDKit), with an [example here](https://github.com/openforcefield/openff-toolkit/issues/1126#issuecomment-969712195)
 - the Jensen group's [xyz2mol program](https://github.com/jensengroup/xyz2mol/)
 
+## I'm getting stereochemistry errors when loading a molecule from a SMILES string.
+
+The toolkit does not accept molecules with undefined stereochemistry. Applying parameters based on [direct chemical perception](https://pubs.acs.org/doi/pdf/10.1021/acs.jctc.8b00640) requires knowing the stereochemistry of chiral centers. Molecule representations without defined stereochemistry might not be given all appropriate parameters specified from the force field, particularly valence terms, around the stereocenters.
+
+## I understand the risks and want to load a molecule from a SMILES string that might not fully define stereochemistry.
+
+Pass `allow_undefined_stereo=True` to molecule loading methods like [Molecule.from_smiles](openff.toolkit.topology.Molecule.from_smiles) to downgrade the exception to a warning. As an example, see the "SMILES without stereochemistry" section in the [Molecule cookbook](users/molecule_cookbook).
```

Comment:
Not quite sure this will link well (and I don't think there's a way to link to a section of a Jupyter notebook); I saw `(see our [Core Concepts](users/concepts)` elsewhere and copied it.

Reply:
We user MyST-notebook in the toolkit, so you can use ordinary MyST code in the notebook. The most robust way to do this is to add a reference target to the notebook (inside the markdown cell):

```markdown
(smiles_wo_stereochem)=
### SMILES without stereochemistry
```

And then link to it:

```markdown
[Molecule cookbook](smiles_wo_stereochem)
```

This way, even if the notebook or section is renamed, moved, broken up etc. the link still works as long as the target is left in the right spot. Ordinarily I'd suggest letting MyST fill in the section name (`[](smiles_wo_stereochem)` renders as "SMILES without stereochemistry"), but in this case we want the FAQ to be degrade gracefully on GitHub, so link names should be explicit.

Reply:
Thanks! This is working in the PR builds now.